### PR TITLE
chore(documentation): #417 update user manual

### PR DESCRIPTION
**Pull request by ds-crehm**
resolves eclipse-tractusx/traceability-foss#417

### CODE REVIEW

File: docs/src/docs/user/user-manual.adoc [hunk 1]
```diff
@@ -39,29 +44,30 @@ User accounts: https://portal.int.demo.catena-x.net/documentation/?path=docs%2F0
...
 To login use the credentials provided by the hosting company.
 
  The application is configured with a technical user, which holds the required IRS roles:
 - Creating a policy in IRS -> admin_irs
 - Requesting jobs in IRS -> view_irs
 
-
  For usage of an EDC it is necessary to provide another technical user. Required roles:
- - Identity Wallet Management Role
+ - Identity wallet management role
```
Comment (mkanal) [inline docs/src/docs/user/user-manual.adoc:56]:
```suggestion
 - role:  "Identity Wallet Management"
```

File: docs/src/docs/user/user-manual.adoc [hunk 2]
```diff
@@ -273,7 +334,7 @@ Once the quality investigation is created you will get a pop-up and can directly
 
 === Supplier part details
 
-Clicking on an item in the list opens "Part details" view.
+To open the detail view click on the three dots icon of the desired item and select "View details".
```
Comment (mkanal) [inline docs/src/docs/user/user-manual.adoc:337]:
```suggestion
To open the detail view, click on the three dots icon of the desired item and select "View details".
```

File: docs/src/docs/user/user-manual.adoc [hunk 3]
```diff
@@ -291,14 +352,14 @@ Detailed information on the IDs for the supplier part/batch.
...
-=== Customer Parts
+=== Customer parts
 
 List view of customer parts and batches.
 Customer Parts that are in a quality alert are highlighted yellow.
 
 === Customer part details
 
-Clicking on an item in the list opens "Part details" view.
+To open the detail view click on the three dots icon of the desired item and select "View details".
```
Comment (mkanal) [inline docs/src/docs/user/user-manual.adoc:362]:
```suggestion
To open the detail view, click on the three dots icon of the desired item and select "View details".
```

File: docs/src/docs/user/user-manual.adoc [hunk 4]
```diff
@@ -112,48 +131,48 @@ Supported languages:
...
+=== Parts table
 
-A Table View split into the different BOM Lifecycle phases (AsBuilt / AsPlanned) of a part.
+A table view split into the different BOM lifecycle phases (AsBuilt / AsPlanned) of a part. By default, both tables are displayed.
+Adjustment of the view of tables can be done by activating/deactivating the tables via the buttons „AS BUILT“ and „AS PLANNED“.
 
 Between the views, there is a slider to adjust the view to make either the left or the right table more visible.
 Alternatively, you can use the view toggle to adjust the visibility of the tables.
 
 Both tables can be sorted, filtered and searched.
```
Comment (mkanal) [inline docs/src/docs/user/user-manual.adoc:146]:
```suggestion
Both tables can be sorted, filtered, and searched.
```

File: docs/src/docs/user/user-manual.adoc [hunk 5]
```diff
@@ -112,48 +131,48 @@ Supported languages:
...
 Gives detailed information on the assets registered in the Digital Twin Registry of Catena-X for the company.
-This includes data based on the aspect models of Use Case Traceability: AsPlanned, SerialPart, Batch.
-Parts that are in a quality alert are highlighted yellow.
+This includes data based on the aspect models of use case traceability: AsPlanned, SerialPart, Batch.
+Parts that have a quality alert are highlighted yellow.
 
-=== Parts selection -> Create Quality alert
+=== Parts selection -> Create quality alert
 
 Select one or multiple child components/parts/batches that are build into your part.
```
Comment (mkanal) [inline docs/src/docs/user/user-manual.adoc:169]:
```suggestion
Select one or multiple child components/parts/batches that are built into your part.
```

File: docs/src/docs/user/user-manual.adoc [hunk 6]
```diff
@@ -232,7 +248,7 @@ Zooming in/out can be done with the corresponding control buttons.
 image:https://raw.githubusercontent.com/eclipse-tractusx/traceability-foss/main/docs/src/images/arc42/user-guide/open-new-tab.png[] Open part tree in new tab to zoom, scroll and focus in a larger view.
 A minimap on the bottom right provides an overview of the current position on the part tree.
```
Comment (mkanal) [inline docs/src/docs/user/user-manual.adoc:249]: There is no minimap IMHO

**Review by mkanal — CHANGES_REQUESTED** [review]

**Review by mkanal — APPROVED** [review]

File: docs/src/docs/user/user-manual.adoc (hunk 6 above)
Comment (ds-crehm) [reply docs/src/docs/user/user-manual.adoc:249]: There is a minimap in the detail view of the parts. Will mark this as resolved.

**Review by mkanal — APPROVED** [review]
LGFM

**Pull request merged**